include/mbedtls/debug.h

```diff
@@ -82,6 +82,17 @@
 
```

Comment:
Typo in commit message: "This commits adds". Also, the description is a bit strange... "Current and end position or the available space as the first argument(s)" - so, how do you pass available space to it?
Also, this macro does not _throw_ anything, it returns an error.

Reply:
What is the typo and suggested fix? I'm sorry I don't see it.

Reply:
`This commit**s** adds` -> `This commit adds`.

Reply:
But looking at the commit history, I'm not sure if this commit is needed at all, since the code from it is removed afterwards.

Reply:
I agree. However, I am trying to remain true to Hanno's artistic vision for his commits by not refactoring them too much.

I'll fix the "This commits" typos as well as the "throw" misuse.